pipelines/nextflow/workflows/nextflow.config

```diff
@@ -62,6 +62,7 @@ process {
 
     withLabel: 'local' {
         executor = 'local'
+        maxForks = 1
```

Comment:
A naive question, if we have a mix of big and small genomes. Having this parameter in the config as 1, will it make the process slow?

Reply:
The local should only be applied for very small tasks (small number of files, all small, lightweight etc.)

If a process resources will vary depending on the size of the genomes, then it should use the variable labels (e.g. "adaptive")